src/test/regress/expected/failure_real_time_select.out

```diff
@@ -0,0 +1,198 @@
+-- 
+--  Failure tests for real time select queries
+-- 
+CREATE SCHEMA real_time_select_failure;
+SET search_path TO 'real_time_select_failure';
+SET citus.next_shard_id TO 190000;
+-- Preparation
+SELECT citus.mitmproxy('conn.allow()');
+ mitmproxy 
+-----------
+ 
+(1 row)
+
+SET citus.shard_count to 4;
+SET citus.shard_replication_factor to 1;
+-- create tables
+CREATE TABLE test_table(id int, value_1 int, value_2 int);
+SELECT create_distributed_table('test_table','id');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+-- Populate data to the table
+INSERT INTO test_table VALUES(1,1,1),(1,2,2),(2,1,1),(2,2,2),(3,1,1),(3,2,2);
+-- Kill when the first COPY command arrived, since we have a single placement 
+-- it is expected to error out.
+SELECT citus.mitmproxy('conn.onQuery(query="^COPY").kill()');
+ mitmproxy 
+-----------
+ 
+(1 row)
+
+SET client_min_messages TO ERROR;
+DO $$
```

Comment:
Can't we make this a SQL function and pass a query to it? We're repeating it many times. And we'll keep copy & pasting it whenever we need it. 
```SQL
CREATE FUNCTION discard_task_id_from_error(query text)
....

```